framework/src/application.ts

```diff
@@ -140,7 +139,6 @@ export class Application {
 			config.label ?? `lisk-${config.genesisConfig?.communityIdentifier}`;
 
 		const mergedConfig = objects.mergeDeep({}, appConfig, config) as ApplicationConfig;
```

Comment:
i think in this file there are place that uses `rootPath` directly

Reply:
I believe whenever rootPath is directly passed the inner function either logger or controller uses the system dir to get the correct path. So that will work.